presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -88,12 +97,23 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+Hadoop catalog
+^^^^^^^^^^^^^^
+
+Iceberg connector supports Hadoop as Catalog
+
 .. code-block:: none
 
     connector.name=iceberg
-    iceberg.catalog.type=nessie
-    iceberg.catalog.warehouse=/tmp
-    iceberg.nessie.uri=https://localhost:19120/api/v1
+    hive.metastore.uri=hostname:port
+    iceberg.catalog.type=hadoop
+
+.. note::
+
+    To use Hadoop catalog with Iceberg table, it requires the ``hive.metastore.uri`` property, even
+    though it doesn't rely on metastore for metadata when using Hadoop Catalog. You can provide
+    a placeholder with ``hive.metastore.uri=thrift://localhost:9083``. This will be fixed in a
```

Comment:
YOu can provide a placeholder with ...
-->
You can provide any value to this property, for example, ``hive.metastore.uri=thrift://localhost:9083``.